api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -192,7 +191,7 @@ private void insertPatientStubIfNeeded(Patient patient) {
 		criteriaExactMatch = new PatientSearchCriteria(sessionFactory, criteriaExactMatch).prepareCriteria(query, true,
 		    true, includeVoided);
 		
-		Set<Patient> patients = new LinkedHashSet<Patient>();
+		Set<Patient> patients = new LinkedHashSet<>();
```

Comment:
@dkayiwa, Its a new feature introduced by Java 7 know as the "Diamond" operator. For more info, follow http://openjdk.java.net/projects/jdk7/features/#f618